cmake/FindFCL.cmake

```diff
@@ -17,15 +17,30 @@ find_package(PkgConfig QUIET)
 pkg_check_modules(PC_FCL fcl QUIET)
 
 # Include directories
-find_path(FCL_INCLUDE_DIRS
-    NAMES fcl/collision.h
-    HINTS ${PC_FCL_INCLUDEDIR}
-    PATHS "${CMAKE_INSTALL_PREFIX}/include")
+if (${PC_FCL_VERSION} VERSION_LESS 0.6.0)
+  find_path(FCL_INCLUDE_DIRS
+      NAMES fcl/collision.h
+      HINTS ${PC_FCL_INCLUDEDIR}
+      PATHS "${CMAKE_INSTALL_PREFIX}/include")
+else()
+  find_path(FCL_INCLUDE_DIRS
+      NAMES fcl/narrowphase/collision.h
+      HINTS ${PC_FCL_INCLUDEDIR}
+      PATHS "${CMAKE_INSTALL_PREFIX}/include")
+endif()
```

Comment:
Thank you for the clarification! That makes sense. It would be good then to add a note of the summary for a future reference. :smile: